pyproject.toml

```diff
@@ -6,6 +6,49 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+authors = [
+    {name = "Kelly Boothby", email = "boothby@dwavesys.com"},
```

Comment:
can't help but notice this duplicates the corresponding entries in `minorminer/package_info.py`

Reply:
Yeah, I tried to read them dynamically, but it doesn't like the format. They haven't changed in years so IMO it's OK to duplicate. At some point we should remove all but the version from `package_info.py`. Our new packages don't include anything except the version there.

Reply:
For posterity, the list of attributes that can by dynamic are determined by `setuptools`. [Here is the current list](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata). Only `version` can be specified as an attribute.